Sources/Networking/Responses/CustomerInfoResponse.swift

```diff
@@ -49,7 +49,7 @@ extension CustomerInfoResponse {
 
         @IgnoreDecodeErrors<PeriodType>
         var periodType: PeriodType
-        var purchaseDate: Date?
+        var purchaseDate: Date
```

Comment:
Just checking that we know this is always available, and having it optional was being too defensive?

Reply:
I checked in the backend and in Android and it's not optional there